src/UserAction/Generic.php

```diff
@@ -34,6 +34,15 @@ class Generic
     /** @var string by default - action is for a single-record */
     public $scope = self::SINGLE_RECORD;
 
+    /** Defining action modifier */
+    const CREATE = 'create'; // create new record(s).
```

Comment:
What is the purpose? If filtering/detection for some specific action set before actual execution, this is probably needed. Otherwise hook can be used with info even like what has been done.

If this is needed, declare as `public const` and rename/prefix all with `MODIFIER_`.

Reply:
done!